demosplan/DemosPlanCoreBundle/Logic/Statement/StatementService.php

```diff
@@ -2243,6 +2243,10 @@ public function createElementsGroupStructureBobHH(string $procedureId, array $st
 
         $noManualSortElementsIds = $this->serviceElements->getHiddenElementsIdsForProcedureId($procedureId);
 
+        $this->statementEntityGrouper->sortSubgroupsAtAllLayers(
+            $group,
+            new TitleGroupsSorter()
+        );
```

Comment:
I was under the impression that at least the "Kapitel" are to be sorted by title, which is not the case if we look at the bug ticket closely. They are to be displayed with the order defined inside their corresponding planning category detail view.